2022-03-15 21:11:46,513 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:955] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 21:29:39,189 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 21:54:18,952 - G:\毕设项目\API\appTest\pc_api\views.py[line:62] - ERROR: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
2022-03-15 21:54:18,952 - G:\毕设项目\API\appTest\pc_api\views.py[line:62] - ERROR: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
2022-03-15 21:56:38,794 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 21:57:30,652 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:04:20,288 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:06:15,906 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:11:59,673 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:999] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:12:31,414 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:999] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:12:41,420 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:999] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:13:27,532 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:999] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:13:50,505 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:999] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:15:26,502 - G:\毕设项目\API\appTest\pc_api\views.py[line:62] - ERROR: 'bool' object has no attribute 'get'
2022-03-15 22:15:26,502 - G:\毕设项目\API\appTest\pc_api\views.py[line:62] - ERROR: 'bool' object has no attribute 'get'
2022-03-15 22:18:06,015 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:18:16,347 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:18:46,447 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:19:40,227 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:19:51,347 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2003, "Can't connect to MySQL server on '120.77.44.219' (timed out)")
2022-03-15 22:41:25,719 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 22:41:25,719 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 22:42:30,260 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 22:42:30,260 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 22:43:30,300 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 22:43:30,300 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-30,30' at line 14")
2022-03-15 23:05:14,912 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '{examListTuple} AND isUse = 1 ) a \n                    ) \n                    AN' at line 10")
2022-03-15 23:05:14,912 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '{examListTuple} AND isUse = 1 ) a \n                    ) \n                    AN' at line 10")
2022-03-15 23:05:53,520 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '{examListTuple} AND isUse = 1 ) a \n                    ) \n                    AN' at line 10")
2022-03-15 23:05:53,520 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:1076] - ERROR: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '{examListTuple} AND isUse = 1 ) a \n                    ) \n                    AN' at line 10")
2022-03-15 23:10:48,068 - G:\毕设项目\API\appTest\utils\dbV3\db.py[line:996] - ERROR: (2013, 'Lost connection to MySQL server during query')
2022-03-15 23:58:21,835 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-15 23:58:21,835 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-15 23:58:24,134 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-15 23:58:24,134 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-16 00:01:17,844 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-16 00:01:17,844 - G:\毕设项目\API\appTest\pc_api\views.py[line:80] - ERROR: 'list' object has no attribute 'update'
2022-03-16 00:23:16,544 - G:\毕设项目\API\appTest\pc_api\views.py[line:39] - ERROR: 'list' object has no attribute 'get'
2022-03-16 00:23:16,544 - G:\毕设项目\API\appTest\pc_api\views.py[line:39] - ERROR: 'list' object has no attribute 'get'
